frontend/src/components/to_the_top.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router";
+
+class ScrollToTop extends Component {
+  componentDidUpdate(prevProps) {
+    if (this.props.location.pathname !== prevProps.location.pathname) {
```

Comment:
hmm maybe we want to always pop to the top even if you are on that page... but good for now 👍